ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/ethernet-interfaces.md

```diff
@@ -977,6 +984,12 @@ interface Ethernet66
    vrrp 3 ipv4 100.64.0.1
    vrrp 3 ipv4 version 3
 !
+interface Ethernet67
+   description Custom_Transceiver_Frequency
+   no shutdown
+   switchport
+   transceiver frequency 190050 ghz
```

Comment:
It seems like the ghz keyword is optional and may depend on platform. I think we need an optional `frequency_unit` key.

```sh
smv031(config-s-clauys-if-Et33/1)#transceiver frequency 1900 ?
% Unrecognized command
smv031(config-s-clauys-if-Et33/1)#transceiver frequency 1900?
% Unrecognized command
smv031(config-s-clauys-if-Et33/1)#transceiver frequency 190000 ?
  ghz   Units in gigahertz
  <cr>  

smv031(config-s-clauys-if-Et33/1)#transceiver frequency 190000 
smv031(config-s-clauys-if-Et33/1)#show sess diff
--- system:/running-config
+++ session:/clauys-session-config
interface Ethernet33/1
+   transceiver frequency 190000.000
smv031(config-s-clauys-if-Et33/1)#transceiver frequency 190000 ghz
smv031(config-s-clauys-if-Et33/1)#show sess diff
--- system:/running-config
+++ session:/clauys-session-config
interface Ethernet33/1
+   transceiver frequency 190000.000 ghz
```

Also we should try to always print with min three decimals. We have an example of that under poe watts

```
{%             set poe_limit_cli = 'poe limit ' ~ "%.2f" | format(ethernet_interface.poe.limit.watts | float) ~ ' watts' %}
```